hs/src/Reach/Verify/SMT.hs

```diff
@@ -1429,6 +1431,14 @@ _smtDefineTypes smt ts = do
   mapM_ type_name ts
   readIORef tmr
 
+smt_export :: DLinExportVal LLBlock -> App SExpr
+smt_export = \case
+  DLEV_Arg at a  -> smt_a at a
+  DLEV_LArg at a -> smt_la at a
+  DLEV_Fun at args body -> do
+    forM_ args $ flip (pathAddUnbound at) O_Export . Just
+    smt_block body
```

Comment:
As mentioned, this will fail if there are loops, because you haven't unrolled them